Userland/Libraries/LibWeb/SVG/SVGTextContentElement.cpp

```diff
@@ -30,18 +30,28 @@ JS::ThrowCompletionOr<void> SVGTextContentElement::initialize(JS::Realm& realm)
     return {};
 }
 
+static float safe_parse_coordinate(DeprecatedString const& value, float fallback)
+{
+    auto coordinate = AttributeParser::parse_coordinate(value);
+    if (coordinate.has_value()) {
+        return coordinate.value();
+    }
+    dbgln("Failed to parse SVG text content coordinate value: {}", value);
```

Comment:
I don't think we want to print a message whenever coordinate parsing failed because it might not be a problem with our parse but just invalid input.

Reply:
Yeah, I was on the fence myself, will remove it.